Sources/RepoToolsCore/RepoActions.swift

```diff
@@ -416,7 +437,7 @@ public enum RepoActions {
                     ),
                 ])
             }
-            fatalError("Cannot extract files other than .zip or .tar")
+            fatalError("Cannot extract files other than .zip, .tar, .tar.gz, or .tgz")
```

Comment:
This isn't actually right, since `.tar` wouldn't be valid.

Reply:
*sigh* you are right, which is really sad considering that this originally said `.tar`...  I will add that suffix check to the list of matches for tarballs.

Reply:
I added `tar` support: https://github.com/discordapp/PodToBUILD/pull/1/commits/43c63479fa365a25e868078b4768303f85124ded#diff-8899a6f8f09176941c964bb6deee3f2cR411